styles/components/_autocomplete.scss

```diff
@@ -0,0 +1,40 @@
+// ==================================================================
+// Autocomplete
+// ==================================================================
+
+////
+/// @group autocomplete
+////
+.sprk-c-Autocomplete__results {
+  list-style-type: $sprk-autocomplete-results-list-style-type;
+  background-color: $sprk-white;
+  border: $sprk-autocomplete-results-border;
+  border-radius: $sprk-autocomplete-results-border-radius;
+  box-shadow: $sprk-autocomplete-results-shadow;
+  max-width: $sprk-autocomplete-results-max-width;
+  max-height: $sprk-autocomplete-results-max-height;
```

Comment:
What do we need a max-height for?

Reply:
This is so the list of results is never taller than a certain height. If we have 100 items in the list we don't want to display all 100 - we just want to show the top 5 and the user can scroll to see the rest.

Reply:
That sounds good.